src/AWSCredentials.jl

```diff
@@ -254,27 +254,62 @@ using IniFile
 dot_aws_credentials_file() = get(ENV, "AWS_CONFIG_FILE",
                                  joinpath(homedir(), ".aws", "credentials"))
 
+dot_aws_config_file() = get(ENV, "AWS_CONFIG_FILE",
+                                 joinpath(homedir(), ".aws", "config"))
+
 """
 Load Credentials from [AWS CLI ~/.aws/credentials file]
 (http://docs.aws.amazon.com/cli/latest/userguide/cli-config-files.html).
 """
-
 function dot_aws_credentials()
 
     @assert isfile(dot_aws_credentials_file())
+    @assert isfile(dot_aws_config_file())
```

Comment:
We need to support systems with only one of these files. So the assert should probably be  creds `||` conf.

Reply:
Up in `function AWSCredentials()` where `dot_aws_credentials()` is called we'll need to apply the same test.